kvdb/backend_js.go

```diff
@@ -0,0 +1,48 @@
+package kvdb
```

Comment:
nit: should we also add `// +build js` here?

Reply:
Ah, this is a naming convention of golang. If the filename matches `_<os_or_architecture>.go` then that implies a build tag. For example if you look at `healthcheck/diskcheck_solaris.go` for example, that doesn't have its own build tag.

Reply:
oh nice, TIL! :)